### PR TITLE
Use nvidia-ml-py and psutil to replace original command line call

### PR DESCRIPTION
**Pull request by Stonesjtu**
I manually tested this branch on my GTX 1080 and GT 650. No older chips are available for me.
The test case won't work under such circumstance now. I'm not familiar with unit-test, so somebody willing to help with the test?

### CODE REVIEW

File: gpustat.py [hunk 1]
```diff
@@ -37,9 +41,11 @@ class ANSIColors:
     BOLD_GREEN  = '\033[1;32m'
     BOLD_YELLOW = '\033[1;33m'
 
-    @staticmethod
-    def wrap(color, msg):
-        return (color + msg + ANSIColors.RESET)
+NOT_SUPPPORTED = 'Not Supported'
+
+@staticmethod
```
Comment (Stonesjtu) [inline gpustat.py]: The wrap is moved from class `ANSIColors` to outside incidentally, which doesn't cause any bugs in my test case. Frustrating~

**Comment by wookayin** [comment]
Thanks for your contribution. I am also keeping my eye on this.

Since the travis CI environment is not equipped with nvidia, current mocks are broken; it makes automatic test a bit complicated.

File: gpustat.py (hunk 1 above)
Comment (wookayin) [inline gpustat.py]: Just because `wrap` is not being used anymore.

**Comment by Stonesjtu** [comment]
The reason why I use `blessings` for color styles is that it will automatically drop ANSI format characters when no TTY is attached (e.g. pipe the output to log file.). And it's more flexible for more styles and theme customization.

**Comment by wookayin** [comment]
The `nvidia-ml-py` package from pypi (not [this fork](jonsafari/nvidia-ml-py)) doesn't support python 3 (`L1831: print c_count.value`). So we need to work around this issue, say to install a modified version (from jonsafari's), etc. Any good idea?

**Comment by wookayin** [comment]
Some commits of mine, on top of your work, have been added in PR #20.

**Comment by Stonesjtu** [comment]
Well I'll close this PR.

**Comment by wookayin** [comment]
Thank you for your contribution, by the way!

**Pull request closed without merging**